pkg/virt-handler/virtwrap/manager.go

```diff
@@ -416,3 +427,125 @@ func (l *LibvirtDomainManager) KillVM(vm *v1.VM) error {
 	l.recorder.Event(vm, kubev1.EventTypeNormal, v1.Deleted.String(), "VM undefined")
 	return nil
 }
+
```

Comment:
@rmohr @admiyo that looks great, though I'd suggest a slight modification:

```
virtwrap/manager.go #interface
virtwrap/domainmanager/libvirt.go # libvirt impl in libvirt module
virtwrap/domainmanager/vdsm.go # vdsm impl in vdsm module 
```

What do you think?